src/components/Todo.tsx

```diff
@@ -14,27 +14,18 @@ interface ListProps {
 }
 
 const Todo: React.FC<ListProps> = ({list}) => { //{list} would replace props so below doesn't look like props.list
-
     const [ toDoList, setToDoList ] = useState(list);
-
     const addTask = (userInput: any ) => {
-        if(!userInput) {
-            alert("Please enter a task");
-            return false;
-        }
-        else {
-            let copyList = [...toDoList];
-            copyList = [...copyList, { id: toDoList.length + 1, description: userInput }];
-            setToDoList(copyList);
-        }
+        let copyList = [...toDoList];
+        copyList = [...copyList, { id: toDoList.length + 1, description: userInput }];
```

Comment:
could set these properties in the parent component that uses todo. this would then mean that we could get rid of internal state for these component and simplify it